artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/Message.java

```diff
@@ -706,9 +707,10 @@ default long getWholeMessageSize() {
 
    /**
     * @return Returns the message in Map form, useful when encoding to JSON
+    * @param valueSizeLimit
     */
-   default Map<String, Object> toMap() {
```

Comment:
This is a breaking api change on core. This is exposed to users

Reply:
the override will sort that and split usage between those who care and those who don't.

Reply:
Yes but this is in interface, we have to keep toMap()

Reply:
yep. sorted. toMap is now untouched.

Reply:
oops miss-read sorry, re read it and youre sorting it, cool.